app/src/main/java/nonameyetsoft/com/torch/MainActivity.java

```diff
@@ -79,4 +65,20 @@ private void initializeXmlReferences() {
         layout = (RelativeLayout) findViewById(R.id.mainLayout);
         switcher = (Switch) findViewById(R.id.switcher);
     }
-}
+
+    @Override
```

Comment:
Is method ko ooper lay ja, ye @Override hai is liye saaray override k doosray k sath honay chahiye. Baaki thek hai.